packages/charts/src/charts/line/components/lines/lines.component.tsx

```diff
@@ -112,7 +110,7 @@ const Lines = ({
               areaMode ? 'area' : 'line'
             }`}
             d={d}
-            variants={createLineMotion(color)}
+            variants={createLineMotion()}
```

Comment:
As line motion is right now static - we can refactor it to `Object` instead of `function`

Reply:
Refactored